lib/rails_translation_manager.rb

```diff
@@ -5,6 +5,8 @@
 require 'rails-i18n'
 
 require 'rails_translation_manager/locale_checker/locale_checker_helper'
+require 'rails_translation_manager/locale_checker/missing_foreign_locales'
+require 'rails_translation_manager/locale_checker/missing_english_locales'
```

Comment:
Don't think I can I'm afraid as the ordering in which the files are required is actually important. For example, if a file is required which references another class that isn't already loaded, then a missing class error is thrown.

Reply:
Ah, fair enough. Though you could (should?) also have the necessary `require` statements in the files themselves.

One of my bugbears is trying to run a test in isolation and finding that it fails because the class I'm testing doesn't pull in something it depends on, and instead I have to run a test suite that handles all that. I think we usually avoid adding `require` calls in Rails because it does a lot of magic for you, but for straight Ruby libraries like this, it's best to have the `require`s in the files where they're needed.

Reply:
I think this was answered in a previous comment but just to address this too. The reason I like to have the files in a central place is to reduce clutter in the files themselves (trying to be all Railsey 😄), this also applies to the specs who may call off to multiple classes. I do understand your point though, but in this instance I don't think the situation you describe would happen as all the [centrally defined files](https://github.com/alphagov/rails_translation_manager/pull/12/files#diff-3991958ab039e4072c5bf831d9ab40a0c932c0d6bee7edd395e43ea5cbd79dffR7-R14) are included throughout the app and the [specs](https://github.com/alphagov/rails_translation_manager/pull/12/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R3). All a spec has to do is import the helper and not worry about any other requiring!